ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.1.yaml

```diff
@@ -0,0 +1,33 @@
+tag_specification:
```

Comment:
I'd say we don't need CI for 4.1 on this branch. We tested upgrades on the 1.3 branch because the last supported release on OCP 4.1 was Serverless 1.2 and we just tested the upgrade from that release. But for the master branch and future Serverless releases we don't have to test it. WDYT?

Reply:
Good call, removed it.